api/test/integration/projects/create-projects.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { projectsContract } from '@shared/contracts/projects.contract';
+import { CreateProjectDto } from '@shared/dtos/projects/create-project.dto';
+import { ACTIVITY } from '@shared/entities/activity.enum';
+import { ECOSYSTEM } from '@shared/entities/ecosystem.enum';
+import { ProjectScorecard } from '@shared/entities/project-scorecard.entity';
+import { Project, PROJECT_PRICE_TYPE } from '@shared/entities/projects.entity';
+import { TestManager } from 'api/test/utils/test-manager';
+import { TestUser } from 'api/test/utils/user.auth';
+
+describe('Create projects', () => {
```

Comment:
We have to clearly note down somewhere that this is only a scaffolding, this can't be considered tested
(we don't really know for sure what to test here as you know)

But yeah, just to keep in mind